pm/queue.go

```diff
@@ -164,3 +167,13 @@ func isNonRetryableTicketErr(err error) bool {
 	// The latter check depends on logic in eth.client.CheckTx()
 	return err == errIsUsedTicket || strings.Contains(err.Error(), "transaction failed")
 }
+
+func (q *ticketQueue) isRecipientActive(addr ethcommon.Address) bool {
+	isActive, err := q.store.IsOrchActive(addr, q.tm.LastInitializedRound())
+	if err != nil {
+		glog.Errorf("Unable to select an active orchestrator")
+		// In the case of error, assume recipient is active in order to try to redeem the ticket
```

Comment:
Is the idea here to bias towards trying to redeem if there is an error with the DB because that's preferable to not redeeming since gas estimation for the tx should still prevent the tx from going on-chain even if the recipient actually happens to not be active?

Reply:
Yes, it's a corner case that the DB returns an error, but still... 

I think it makes more sense in uncertain scenarios to try to redeem the ticket because it may expire.